Change all issue templates to github issue forms

bappitybup opened the pull request:
- Removed old .md issue templates (except questions / discussion)
- Added new issue form github templates ([syntax examples here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms))
### NEW FEATURE REQUEST ISSUE FORM:
![image](https://github.com/dessalines/jerboa/assets/12457072/5c1a9478-2a89-43c0-8f7c-da9225ce441f)
### NEW BUG REPORT ISSUE FORM:
![image](https://github.com/dessalines/jerboa/assets/12457072/933eb90c-1fe1-492d-aab6-8b0fe33eec80)
### in the examples above:
- the summary markdown at the bottom can be viewed and will not be submitted with the issue form
- the issue templates submit the headings and the content entered by users below them (and also the checkboxes)
- some input/textarea/checkbox fields have been made "required" so the submit button can only be pressed when they have content / have been updated
- I have changed the question on the last checkbox to "This is not a question or discussion" rather than "Is this a question or a discussion" so that it can be checked instead of left alone

### Some notes:
- individual checkboxes can be made required or not required so I could change the last checkbox back to how it was and just make it not required.
- I can change the Jerboa Version input field to a dropdown list of versions but then I'd have to update the dropdown list every time a new version is released which seems like a hassle

bappitybup commented:
https://github.com/bappitybup/testrepo/issues/new/choose <-- examples with the issue forms that you can test

dessalines commented on .github/ISSUE_TEMPLATE/bug_report.yaml:
```diff
@@ -0,0 +1,95 @@
+      value: |
+        <details><summary>Click here if you don't know how to get the logs</summary>
+
+
```
Instead of this, there's now a crash logs helper. Tell them to go to `settings -> about -> Crash Logs`

MV-GH replied:
Keep that one, cuz the crash log helper wont help if the crash is on startup. And we have had a handfull of those.

bappitybup replied:
should I just add some text in this summary dropdown for the crash log helper? or replace it altogether?

MV-GH replied:
Tell them to use the previous method if the crash is on startup else use the crash logs

bappitybup replied:
see latest comment in PR, added two sections
![image](https://github.com/dessalines/jerboa/assets/12457072/17e5a683-e6bc-4ba0-ac9a-cbe7d3f56f1b)
**FOR CRASHES ON STARTUP** has the old content
**FOR GENERAL CRASHES** has this:
![image](https://github.com/dessalines/jerboa/assets/12457072/bda79777-42ca-417a-a8a9-4f8b1a2e31e5)

dessalines reviewed (commented):
Thanks a ton!

> individual checkboxes can be made required or not required so I could change the last checkbox back to how it was and just make it not required.

Yeah make them required if you would.

A dropdown for versions isn't necessary.

bappitybup commented:
> Thanks a ton!
> 
> > individual checkboxes can be made required or not required so I could change the last checkbox back to how it was and just make it not required.
> 
> Yeah make them required if you would.
> 
> A dropdown for versions isn't necessary.

no prob 👌 all of them are already required

bappitybup commented:
I have added two sections for startup crashes and general crashes, updated the example as well
--> https://github.com/bappitybup/testrepo/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=bug_report.yaml

MV-GH commented on .github/ISSUE_TEMPLATE/bug_report.yaml:
```diff
@@ -0,0 +1,108 @@
+        > - Connect your phone to your PC
+        > - Execute this command on PC
+        > 
+        > ### To enable developer options and USD DEBUGGING
```
Fix my typo there, and ill approve it
